coil-core/src/nonAndroidMain/kotlin/coil3/Image.kt

```diff
@@ -4,21 +4,45 @@ import coil3.annotation.Data
 import coil3.annotation.ExperimentalCoilApi
 import kotlin.jvm.JvmOverloads
 import org.jetbrains.skia.Bitmap
+import org.jetbrains.skia.Canvas
+import org.jetbrains.skia.svg.SVGDOM
 
 @ExperimentalCoilApi
 @JvmOverloads
 fun Bitmap.asCoilImage(
     shareable: Boolean = isImmutable,
 ): Image = BitmapImage(this, shareable)
 
+@ExperimentalCoilApi
+fun SVGDOM.asCoilImage(
```

Comment:
We should keep this private. Maintaining this public API going forward could be tough.

Reply:
Done